### DISCUSSION TITLE
Why does not the Power::TYPE_TRANSFORM power use anywhere?

### DISCUSSION THREAD

**Discussion by GeosCoin** [flareteam/flare-game#968] [General]
In empyrean_campaign mod, I can not find any power type with TYPE_TRANSFORM wherever in items, magican , ranger, or warrior. 

So I wonder why it did this?

**Comment by dorkster** [comment]
We just didn't see a need for it. The idea of having the player transform into another creature didn't fit the theme of the game we were making.

If you're looking for examples of transform powers, we *did* use it in the alpha demo for the [stealth power](https://github.com/flareteam/flare-game/blob/master/mods/alpha_demo/powers/categories/ranger.txt#L147-L161). And [Polymorphable](https://flarerpg.org/mods/polymorphable/) went even further and used transformations as a way to gate progression (i.e. need to transform into a bat to fly over a river).

**Reply by GeosCoin** [reply]
Great, Thank you! 
That's interesting and I'll try it .  :)
